tools/slicec-cs/src/slicec_ext/member_ext.rs

```diff
@@ -31,6 +31,8 @@ impl<T: Member> MemberExt for T {
 pub trait FieldExt {
     /// Check if this field, or its parent struct, are marked with `cs::readonly`.
     fn is_cs_readonly(&self) -> bool;
+
+    fn formatted_param_doc_comment(&self) -> Option<String>;
```

Comment:
It'd be nice to copy the doc comment on this.
And if you went with the `panic`, also mention this should only be called on enumerators, because those are the only place fields support `@param`.

Reply:
Added doc-comment.